workspace-manager-clienttests/README.md

```diff
@@ -160,4 +161,16 @@ Please refer to `test-runner-integration.yml` for an example use case.
 cd ~/terra-workspace-manager/workspace-manager-client
 ../gradlew clean assemble
 ls -la ./build/libs/*jar
+```
+
+#### Use a local Workspace Manager server
+See [DEVELOPMENT.md](../DEVELOPMENT.md#running-workspace-manager) to start a
+local Workspace Manager server.
+
+Workspace Manager has a [pre-defined configuration file](src/main/resources/servers/workspace-local.json) 
+for running against a local server. Once you have a local server running separately,
+you can run client tests against that server, e.g.
+```
+export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-local.json" 
```

Comment:
Would you ever want to reset this to the "default"? What is the default? Are there other options?